internal/cmd/validate_test.go

```diff
@@ -0,0 +1,174 @@
+package cmd
+
+import (
+	"context"
+	_ "embed"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/logr/funcr"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"gotest.tools/v3/assert"
+
+	"package-operator.run/package-operator/internal/packages/packagecontent"
+)
+
+func TestValidateConfig(t *testing.T) {
+	t.Parallel()
+
+	testLogger := funcr.New(func(_, _ string) {}, funcr.Options{})
```

Comment:
Use https://github.com/go-logr/logr/blob/v1.2.4/testr/testr.go#L29

Reply:
Sure, this can wait, but the longer it sits the more rebases will be required. So for review purposes I'm looking more for feedback on whether the commands still work the way they are intended to. The actual refactor should be fairly straight forward.

Reply:
Right you are, let me do that now